blockchain/My-Wallet-V3-iOS
# feat(Homebrew-Exchange): add label for fiat decimal amount.

## Objective

Add smaller label for fiat decimal amount input.

## Description

~~Added label to right-hand side of the primary amount label and display the decimal value when fiat input is switched on.~~~ EDIT: This will be deprecated using a more accurate approach using attributed text, but this PR still should be merged to avoid conflicts.

`InputsState` is also deprecated and is removed in active branches.

## How to Test

- Checkout 160de9607ccfb74ff9ab2df0ab601d089adc4d77 and open Exchange.
- Type some numbers
- Tap on the right-hand side button (currently text is truncated `...`) to switch to fiat
- Type more numbers after decimal
- Currently known that fiat symbol is missing and limit of 2 decimal places should be enforced for fiat - more changes are coming.

## Screenshot/Design assessment

Not really close to designs at all yet other than amounts being centered properly.
<img width="328" alt="screen shot 2018-09-07 at 4 37 12 pm" src="https://user-images.githubusercontent.com/10579422/45241951-4d94e700-b2bc-11e8-9f54-ce0e04617eb4.png">

## Merge Checklist
